catalog/ftp/src/main/java/ddf/catalog/ftp/FtpServerManager.java

```diff
@@ -252,7 +252,8 @@ private void waitForConnections() {
           break;
         }
       } catch (InterruptedException e) {
-        Thread.interrupted();
+        Thread.interrupted(); // <<<<<<<< Do we still need this line?
+        Thread.currentThread().interrupt();
```

Comment:
Reminder to fix this TODO

Reply:
@emmberk yeah wanted to make sure we addresed this line. Do we need line 225 if we are going to interrupt the currentThread on 256?